.gitignore

```diff
@@ -0,0 +1,43 @@
+*.bak
+*.iml
+*.log
+*.swo
+*.swp
+*/.glide
+*/vendor
+.DS_Store
+.bak
+.idea/
+.revision
+.tendermint
+.tendermint-lite
```

Comment:
The .gitignore file seems to stem from working on core or copied over? Let's not exclude directories that won't exist or might have another meaning when working with this repo.